README.md

```diff
@@ -1,38 +1,68 @@
-[![Build Status][travis-image]][travis-url]
-[![Go Report Card][goreportcard-image]][goreportcard-url]
-[![codecov][codecov-image]][codecov-url]
- 
-## Snaps
-
-Package any app for every Linux desktop, server, cloud or device.
-
-Snaps are faster to install, easier to create, safer to run, and they update
-automatically and transactionally so your app is always fresh and never
-broken. You can bring your own build infrastructure or use ours.
-
-Head over to [snapcraft.io](https://snapcraft.io) to get started.
+[![Snapcraft](https://avatars2.githubusercontent.com/u/19532717?s=200)](https://snapcraft.io)
+
+# Welcome to snapd
+
+This is the code repository for **snapd**, the background service that manages
+and maintains installed snaps. 
+
+Snaps are app packages for desktop, cloud and IoT that update automatically,
+are easy to install, secure, cross-platform and dependency-free. They're being
+used on millions of Linux systems every day.
+
+Alongside its multifarious service and management functions, snapd:
+- implements the confinement policies that isolate snaps from the base system
+  and from each other
+- provides the _snap_ command that's used to install and remove snaps and
```

Comment:
I would put this item first, it's more generically applicable to visitors who don't know about snap confinement and isolation and things, they might just know snaps as that thing with the `snap` command

Reply:
Good idea, thanks.